rs/ic-observability/multiservice-discovery/src/metrics.rs

```diff
@@ -0,0 +1,57 @@
+use opentelemetry::{
+    global,
+    metrics::{Counter, UpDownCounter},
+};
+
+#[derive(Clone)]
+pub struct MSDMetrics {
+    pub definitions: UpDownCounter<i64>,
+    pub running_definition_metrics: RunningDefinitionsMetrics,
+}
+
+impl Default for MSDMetrics {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl MSDMetrics {
+    pub fn new() -> Self {
+        let meter = global::meter("axum-app");
+        let definitions = meter
+            .i64_up_down_counter("msd.definitions.total")
```

Comment:
Not a monotonically increasing counter, so total doesn't apply.  This one can be left at msd.definitions.

Reply:
Idem in other gauges.

https://opentelemetry.io/docs/specs/otel/metrics/api/#updowncounter